tortoise/backends/base/schema_generator.py

```diff
@@ -449,16 +364,104 @@ def _get_table_sql(self, model: "type[Model]", safe: bool = True) -> dict:
                     lines.insert(-1, indent + unique_index_create_sql)
                     m2m_create_string = "\n".join(lines)
             m2m_tables_for_create.append(m2m_create_string)
+        return m2m_tables_for_create
+
+    def _generate_field_default(
+        self, field_object: Field, table_name: str, column_name: str, model: type[Model]
+    ) -> str:
+        auto_now_add = getattr(field_object, "auto_now_add", False)
+        auto_now = getattr(field_object, "auto_now", False)
+        default = field_object.default
+        if default is not None or auto_now or auto_now_add:
+            if not callable(default) and not isinstance(
+                field_object, (UUIDField, TextField, JSONField)
+            ):
+                default = field_object.to_db_value(default, model)
+                try:
+                    return self._column_default_generator(
+                        table_name,
+                        column_name,
+                        self._escape_default_value(default),
+                        auto_now_add,
+                        auto_now,
+                    )
+                except NotImplementedError:
+                    pass
+        return ""
+
+    def _get_table_sql(self, model: type[Model], safe: bool = True) -> dict:
+        fields_to_create = []
+        fields_with_index = []
+        references = set()
+        models_to_create: list[type[Model]] = []
+
+        self._get_models_to_create(models_to_create)
+        table_name = model._meta.db_table
+        models_tables = [model._meta.db_table for model in models_to_create]
+        for field_name, column_name in model._meta.fields_db_projection.items():
+            field_object = model._meta.fields_map[field_name]
+            comment = self._generate_field_comment(field_object, table_name, column_name)
+            default = self._generate_field_default(field_object, table_name, column_name, model)
+
+            # TODO: PK generation needs to move out of schema generator.
+            if create_pk_field := self._generate_pk_create_sql(field_object, column_name, comment):
+                fields_to_create.append(create_pk_field)
+                continue
+
+            field_creation_string, related_table_name = self._generate_field_creation(
```

Comment:
`_generate_field_creation` doesn't seem to be valid English. Should it be something like `_get_field_sql`?

Reply:
Changed to be `_get_field_sql_and_related_table`